src/libs/Log/clientLoggingCallback.js

```diff
@@ -0,0 +1,2 @@
+// On most platforms we just log on the debug channel
+export default console.debug;
```

Comment:
I think you'd have to create a directory called `clientLoggingCallback` w/ `index.js` and `index.desktop.js`, right?

Reply:
I wasn't sure what's the convention, so `.desktop.js` can only be used on `index` files? 
Ok I'll apply the change